plugins/janus_videoroom.c

```diff
@@ -1916,6 +1948,7 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 		packet.timestamp = ntohl(packet.data->timestamp);
 		packet.seq_number = ntohs(packet.data->seq_number);
 		/* Go */
+		janus_mutex_lock(&participant->subscribers_mutex);
```

Comment:
Neither me nor any of the users connecting to the gateway have noticed any delays after this commit was added so this lock seems fine for performance with around 700 handles at a time